taxi/views.py

```diff
@@ -52,3 +52,43 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    template_name = "taxi/manufacturer_form.html"
+    success_url = "taxi/manufacturer_list"
```

Comment:
The `success_url` should be a URL pattern name, not a string representing a path. Consider using Django's `reverse_lazy` to reference the URL pattern name, like `reverse_lazy('manufacturer-list')`.